hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -185,11 +196,9 @@ public BackgroundTaskResult call() throws Exception {
               db.deleteFilesNotMatchingPrefix(prefixPairs, FILTER_FUNCTION);
             }
           }
-
-          // mark the snapshot as filtered by writing to the file
-          String content = snapshotInfo.getSnapshotId() + "\n";
-          Files.write(filePath, content.getBytes(StandardCharsets.UTF_8),
-              StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+          markSSTFilteredFlagForSnapshot(snapshotInfoTable,
```

Comment:
do we need snapshotInfoTable as a parameter ?

Reply:
updated it. Done.